lint-workflow/lint.py

```diff
@@ -11,50 +11,53 @@ def get_action_update(action_id):
+        headers = {"user-agent": "bw-linter"}
 
-    if os.getenv("GITHUB_TOKEN", None):
-        headers["Authorization"] = f"Token {os.environ['GITHUB_TOKEN']}"
+        if os.getenv("GITHUB_TOKEN", None):
+            headers["Authorization"] = f"Token {os.environ['GITHUB_TOKEN']}"
 
-    if "bitwarden" in path:
-        path_list = path.split("/", 2)
-        url = f"https://api.github.com/repos/{path_list[0]}/{path_list[1]}/commits?path={path_list[2]}"
-        r = http.request("GET", url, headers=headers)
-        sha = json.loads(r.data)[0]["sha"]
-        if sha not in hash:
-            return f"https://github.com/{path_list[0]}/{path_list[1]}/commit/{sha}"
-    else:
-        # Get tag from latest release
-        r = http.request(
-            "GET",
-            f"https://api.github.com/repos/{path}/releases/latest",
-            headers=headers,
-        )
-
-        tag_name = json.loads(r.data)["tag_name"]
-
-        # Get the URL to the commit for the tag
-        r = http.request(
-            "GET",
-            f"https://api.github.com/repos/{path}/git/ref/tags/{tag_name}",
-            headers=headers,
-        )
-        if json.loads(r.data)["object"]["type"] == "commit":
-            sha = json.loads(r.data)["object"]["sha"]
-        else:
-            url = json.loads(r.data)["object"]["url"]
-            # Follow the URL and get the commit sha for tags
+        if "bitwarden" in path:
+            path_list = path.split("/", 2)
+            url = f"https://api.github.com/repos/{path_list[0]}/{path_list[1]}/commits?path={path_list[2]}"
             r = http.request("GET", url, headers=headers)
-            sha = json.loads(r.data)["object"]["sha"]
+            sha = json.loads(r.data)[0]["sha"]
+            if sha not in hash:
+                return f"https://github.com/{path_list[0]}/{path_list[1]}/commit/{sha}"
+        else:
+            # Get tag from latest release
+            r = http.request(
+                "GET",
+                f"https://api.github.com/repos/{path}/releases/latest",
+                headers=headers,
+            )
+
+            tag_name = json.loads(r.data)["tag_name"]
 
-        if sha not in hash:
-            return f"https://github.com/{path}/commit/{sha}"
+            # Get the URL to the commit for the tag
+            r = http.request(
+                "GET",
+                f"https://api.github.com/repos/{path}/git/ref/tags/{tag_name}",
+                headers=headers,
+            )
+            if json.loads(r.data)["object"]["type"] == "commit":
+                sha = json.loads(r.data)["object"]["sha"]
+            else:
+                url = json.loads(r.data)["object"]["url"]
+                # Follow the URL and get the commit sha for tags
+                r = http.request("GET", url, headers=headers)
+                sha = json.loads(r.data)["object"]["sha"]
+
+            if sha not in hash:
+                return f"https://github.com/{path}/commit/{sha}"
+    except:
```

Comment:
Can we log something here, maybe the error that was caught? Probably just a print statement, since we haven't implemented logging yet.